content/docs/rfcs/44/README.md

```diff
@@ -0,0 +1,267 @@
+We refer to the former as a stem message and to the latter as a fluff message.
+A message starts in stem phase, and at some point, transitions to fluff phase.
+Nodes, on the other hand, are in stem state or fluff state.
+Nodes in stem state relay stem messages to a single, per epoch and incoming stem connection randomly selected, relay node.
+Nodes in fluff state transition stem messages into fluff phase and relay them accordingly.
+Fluff messages are always disseminated via Waku Relay (with added random delay).
+
+Messages originated in the node (i.e. messages coming from the application layer of our node),
+are always sent as stem messages.
+
+The stem phase can be seen as a different protocol, and messages are introduced into Waku Relay, and by extension gossipsub,
+once they arrive at a node in fluff state for the first time.
+44/WAKU2-DANDELION uses [19/WAKU2-LIGHTPUSH](/spec/19/) as the protocol for relaying stem messages.
+
+There are no negative effects on gossipsub peer scoring,
+because Dandelion nodes in *stem state* still normally relay Waku Relay (gossipsub) messages.
+
+# Specification
+
+Nodes supporting 44/WAKU2-DANDELION MUST either be in stem state or in fluff state.
+This does not include relaying messages originated in $v$, for which $v$ SHOULD always be in stem state.
+
+## Choosing the State
+
+On startup and when a new epoch starts,
+node $v$ randomly selects a number $r$ between 0 and 1.
+If $r < q$, for $q = 0.2$, the node enters fluff state, otherwise, it enters stem state.
+
+New epochs start when `unixtime` (in seconds) $\equiv 0 \mod 600$,
+corresponding to 10 minute epochs.
+
+## Stem State
+
+On entering stem state,
+nodes supporting 44/WAKU2-DANDELION MUST randomly select two nodes for each pubsub topic from the respective gossipsub mesh node set.
+These nodes are referred to as stem relays.
+Stem relays MUST support [19/WAKU2-LIGHTPUSH](/spec/19/).
+If a chosen peer does not support [19/WAKU2-LIGHTPUSH](/spec/19/),
+the node SHOULD switch to fluff state.
+(We may update this strategy in future versions of this document.)
+
+Further, the node establishes a map that maps each incoming stem connection
+to one of its stem relays chosen at random (but fixed per epoch).
+Incoming stem connections are identified by the [Peer IDs](https://docs.libp2p.io/concepts/peers/#peer-id/)
+of peers the node receives [19/WAKU2-LIGHTPUSH](/spec/19/) messages from.
+The node itself is mapped in the same way, so that all messages originated by the node are relayed via a per-epoch-fixed Dandelion relay, too.
+
+While in stem state, nodes MUST relay stem messages to the respective stem relay.
+Received fluff messages MUST be relayed as specified in the fluff state section.
+
+The stem protocol ([19/WAKU2-LIGHTPUSH](/spec/19/)) is independent of the fluff protocol ([Waku Relay](/spec/11/)).
+While in stem state, nodes MUST NOT gossip about stem messages,
+and MUST NOT send control messages related to stem messages.
```

Comment:
Right. Trying to think if this requires altering underlying gossipsub behaviour in any way, but nodes will only start gossiping about messages once a node publishes the message on the `pubsubTopic` and it enters the fluff phase.
(Thinking out loud :) )